src/tools/ceph_monstore_tool.cc

```diff
@@ -532,6 +532,13 @@ static int update_auth(MonitorDBStore& st, const string& keyring_path)
     KeyServerData::Incremental auth_inc;
     auth_inc.name = k.first;
     auth_inc.auth = k.second;
+    if (auth_inc.auth.caps.empty()) {
```

Comment:
Generally:
generating the ceph.client.admin.keyring  when we deploy ceph, and no caps info in the ceph.client.admin.keyring.
so fill auth.cap with the default caps if it is empty, whether in favor of the use of the tool?